src/browser/browser.js

```diff
@@ -107,6 +100,14 @@ module.exports = class Browser {
         }, {});
     }
 
+    _collectCustomCommands() {
+        this._session.overwriteCommand("addCommand", (origCommand, name, wrapper, elementScope) => {
```

Comment:
> Do you mean that new arguments may appear in the near future?

User might specify his own. Like we did for `click`: https://github.com/gemini-testing/testplane-safari-commands/blob/hermione/v1.0.0/lib/commands/click.js#L4

Generally it is always better to mimic the original API for wrappers.

I usually use `return origCommand` even for commands with ReturnType: `Promise<void>`, because you might want to change it in the future. For example: https://nda.ya.ru/t/GVBoakzU75StY3

Reply:
> User might specify his own. Like we did for click: https://github.com/gemini-testing/testplane-safari-commands/blob/hermione/v1.0.0/lib/commands/click.js#L4

These arguments passed inside `wrapper` function so in this case everything works correctly.

It can breaks only if someone add custom arguments to the `addCommand`. So I will use `...rest` anyway